apps/protocol-frontend/src/components/MobileNav.tsx

```diff
@@ -110,6 +127,64 @@ const MobileNav = ({ children, isOpen, onClose }: MobileNavProps) => {
                           active={location.pathname.includes('/profile')}
                         />
                       </Link>
+                      {isConnected && isAuthenticated && (
+                        <Stack>
+                          {!daosListIsLoading &&
+                          daosListData &&
+                          daosListData.length > 0 ? (
```

Comment:
It might be good to abstract this to a readable boolean higher up in the component